### ISSUE TITLE
Kernel Size = 5?

### ISSUE THREAD

**Issue by KevinHooah** [wdkhuans/DynamicWHAR#1]
Hi,

In the paper, there is a sentence that mentions the kernel size is 5: "Then, let the output go through a temporal convolution block, which contains 𝐾 1D convolution kernels of size 5, to mine the relations among different time steps of each sensor."

However, all `kernel_size` is set as 1 in the `modules.py`.  May I ask which one should be 5 instead of 1?

Thanks!

**Comment by wdkhuans** [comment]
Hello! You are right. In **line 20** of the `modules.py`, the `kernel_size` of `self.conv2` should be `5`. **Line 20** should be changed to:

> `self.conv2 = nn.Conv1d(self.window_size, self.time_reduce_size, kernel_size=5, stride=1, padding=2)`

 Thank you for the careful discovery :smile:.

**Comment by KevinHooah** [comment]
Looks great! Thank you for the clarification!
